seahub/thumbnail/utils.py

```diff
@@ -6,6 +6,8 @@
 import urllib2
 import logging
 from StringIO import StringIO
+from PIL import Image, ImageCms
+from psd_tools import PSDImage
```

Comment:
sry, i can ask if it is about psd tools or just what I wrote about psd_tools code?